src/main/java/org/cloudfoundry/autosleep/config/Deployment.java

```diff
@@ -7,15 +7,29 @@
 
 import java.util.UUID;
 
-@Getter
-@Setter
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class Deployment {
+
     @JsonProperty("application_id")
+    @Getter
+    @Setter
     private UUID applicationId;
 
     @JsonProperty("application_name")
+    @Getter
+    @Setter
     private String applicationName;
 
+    @JsonProperty("application_uris")
+    private String[] applicationUris;
+
+    public String getFirstUri() {
+        return applicationUris[0];
+    }
+
+    public void setApplicationUris(String[] uris) {
```

Comment:
This would trigger E2 findbugs error:

	
```
EI2: May expose internal representation by incorporating reference to mutable object (EI_EXPOSE_REP2)

This code stores a reference to an externally mutable object into the internal representation of the object.  If instances are accessed by untrusted code, and unchecked changes to the mutable object would compromise security or other important properties, you will need to do something different. Storing a copy of the object is better approach in many situations.
```